packer_variables.json

```diff
@@ -1,5 +1,5 @@
 {
-  "cfncluster_version": "1.4.1",
+  "cfncluster_version": "1.4.2",
   "cfncluster_cookbook_version": "1.4.1",
```

Comment:
did we not have to bump the cookbook version as well?

Reply:
Cookbook is still at 1.4.1 ( CfnCluster v1.4.1 release used cfncluster-cookbook v1.4.0 as-is).

Reply:
ah, ok.